### PR TITLE
Update README.md about installation for Arch Linux

### PR DESCRIPTION
**Pull request by orhun**
Hey!

`ht` is moved to the [community](https://wiki.archlinux.org/index.php/Official_repositories#community) repository of Arch Linux: https://archlinux.org/packages/community/x86_64/ht/
This pull request basically updates the instructions about installating `ht` via [pacman](https://wiki.archlinux.org/index.php/Pacman). (2433cfd)

One little issue is `ht` is installed as `/usr/bin/ht-rs` since it conflicts with [another package](https://archlinux.org/packages/extra/any/texlive-core/) which installs a binary named "ht" to `/usr/bin/ht`. I'm not sure what else can be done about this but I thought it'd appropriate to make you aware of this situation. Users can easily work this around with a simple `alias ht=ht-rs`.

BR,
orhun.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,11 +12,10 @@ Yet another [HTTPie](https://httpie.io/) clone in Rust.
 brew install ht-rust
 ```
 
-### On Arch Linux via AUR
-[ht-bin AUR package](https://aur.archlinux.org/packages/ht-bin)
+### On Arch Linux via Pacman
 
 ```
-yay -S ht-bin
+pacman -S ht
```
Comment (ducaale) [inline README.md:18]: Can you add a comment mentioning the name of the binary that will be installed?

**Comment by ducaale** [comment]
>One little issue is ht is installed as /usr/bin/ht-rs since it conflicts with another package which installs a binary named "ht" to /usr/bin/ht.

I wonder how feasible it is to rename the installed binary from `ht-rs` to `ht-rust`? That would reduce the list of names this project uses to `ht` and `ht-rust`.
* https://github.com/NixOS/nixpkgs/pull/112110
* https://github.com/ScoopInstaller/Main/pull/1800
* https://github.com/Homebrew/homebrew-core/pull/70583

File: README.md (hunk 1 above)
Comment (orhun) [reply README.md:18]: Sure, see b73bd09f3094da6e9900fc822d4b50d387e93c53.

**Comment by orhun** [comment]
I renamed the installed binary to `ht-rust` ([archlinux/svntogit-community/d01b546](https://github.com/archlinux/svntogit-community/commit/d01b5467121e2631f6af5345d8ab2a9ebe98b8d9#diff-3e341d2d9c67be01819b25b25d5e53ea3cdf3a38d28846cda85a195eb9b7203a)) and updated README.md about this. (103f88b)

File: README.md (hunk 1 above)
Comment (orhun) [reply README.md:18]: 103f88b

**Review by ducaale — APPROVED** [review]
Looks good. Thanks, @orhun

**Pull request merged**